package.json

```diff
@@ -19,12 +19,23 @@
     "iso8601-duration": "^1.2.0",
     "node-fetch": "^2.6.0",
     "redis": "^3.0.2",
+    "request": "^2.88.2",
```

Comment:
how was this working before? 🤔

Reply:
I already had it locally, and it must have been installing properly as the build job installs deps cleanly
```
cat node_modules/request/package.json | jq .version
"2.72.0"
```

Reply:
https://github.com/ajayyy/SponsorBlockServer/issues/177